      E1:     C1 CALL pred ho_5.main/2-0 (cc_multi) ho_5.m:14
mdb> echo on
Command echo enabled.
mdb> register --quiet
mdb> break p
 0: + stop  interface pred ho_5.p/2-0 (det)
mdb> continue
      E2:     C2 CALL pred ho_5.p/2-0 (det) ho_5.m:22
mdb> finish
      E3:     C2 EXCP pred ho_5.p/2-0 (det) c2; ho_5.m:24
mdb> dd -d 3 -n 7
Call p(1, _)
Throws zero
Expected? no
q(1, 0)
Valid? yes
Call r(0, _)
Throws zero
Expected? yes
Found unhandled or incorrect exception:
p(1, _)
zero
Is this a bug? yes
      E3:     C2 EXCP pred ho_5.p/2-0 (det) c2; ho_5.m:24
mdb> continue
mdb: warning: reached unknown label
This may result in some exception events
being omitted from the trace.
exception(univ_cons('<<function>>'))
      E4:     C3 CALL pred ho_5.p/2-0 (det) ho_5.m:22
mdb> finish
      E5:     C3 EXCP pred ho_5.p/2-0 (det) c2; ho_5.m:24
mdb> dd -d 3 -n 7
Call p(2, _)
Throws zero
Expected? no
q(2, 0)
Valid? yes
Found unhandled or incorrect exception:
p(2, _)
zero
Is this a bug? yes
      E5:     C3 EXCP pred ho_5.p/2-0 (det) c2; ho_5.m:24
mdb> continue
mdb: warning: reached unknown label
This may result in some exception events
being omitted from the trace.
exception(univ_cons('<<function>>'))
